crates/forge-runner/src/forge_config.rs

```diff
@@ -0,0 +1,76 @@
+use std::num::NonZeroU32;
+use std::sync::Arc;
+
+#[derive(Debug, PartialEq)]
+pub struct ForgeConfig {
+    pub runner_config: Arc<RunnerConfig>,
+    pub runtime_config: Arc<RuntimeConfig>,
```

Comment:
I don't understand the difference between those.

Reply:
`RuntimeConfig` is a config of the execution runtime (running Cairo) - those are e.g. vm constraints like max steps or info about the need of collection of vm trace. It related to cheatnet stuff mostly.
`RunnerConfig` impact on the other hand is strictly limited to `forge-runner` crate as it controls the test flow - exit_first - and fuzzing